src/main/scala/mesosphere/mesos/TaskBuilder.scala

```diff
@@ -22,7 +21,8 @@ class TaskBuilder(
     runSpec: RunSpec,
     newTaskId: PathId => Task.Id,
     config: MarathonConf,
-    appTaskProc: Option[RunSpecTaskProcessor] = None) {
+    appTaskProc: Option[RunSpecTaskProcessor] = None,
+    rejectionCollectorOpt: Option[RejectOfferCollector]) {
```

Comment:
Do you think that return rejections collector from TaskBuilder.build is a better idea ?
This might complicate TaskOpFactoryImpl.
Why don't hide creation of RejectOfferCollector in MarathonModule ?